src/Cli/dotnet/commands/dotnet-run/RunCommandParser.cs

```diff
@@ -47,6 +47,8 @@ public static Command GetCommand()
             command.AddOption(InteractiveOption);
             command.AddOption(NoRestoreOption);
             command.AddOption(CommonOptions.VerbosityOption());
+            command.AddOption(CommonOptions.ArchitectureOption());
+            command.AddOption(CommonOptions.OperatingSystemOption());
```

Comment:
Does this work correctly for `dotnet run`?  The output path is going to be different if you specify the OS or architecture, does the run command take that into account?